### PR TITLE
Improve media icons and add audio analytics

### LINKED ISSUE #3123: Speaker, play, pause icons should be more clear in dark mode
#### Software and hardware versions 
Collect v1.18 (older versions were not been verified)

#### Problem description
Speaker, play, pause icons should be more clear in dark mode

#### Steps to reproduce the problem
1. User sets `Dark mode`
2. User fills `Birds` form - [birds.zip](https://github.com/opendatakit/collect/files/3247268/birds.zip)
3. User sees page with `speaker, play, pause` icons

|Screenshot_20190603-130006|Screenshot_20190603-132212|Screenshot_20190603-132234|
|---|---|---|

#### Expected behavior
* A good idea may be to change the background of icons

### PR DESCRIPTION
**Pull request by seadowg**
Closes #3123. Relies on #3353.

One of the motivators for this was a session I spent with a designer recently playing with the audio features in Collect. We came up with a bunch of ideas and questions and for this I've just pulled in the simplest/least controversial. I'll list them here with the changes made:

* Change `AudioButton` icons so that it doesn't make it look like you can "pause" audio
    * Use stop icon instead of pause icon
* Could have more highlighting when audio is playing
    * Highlight `AudioButton` along with text
* Hard to understand difference between audio and video buttons
    * Use "volume" icon for audio and "play on screen icon" for video
* Icons on fast fwd and back (`AudioWidget`) should indicate how much they skip
    * Use Materials built in forward and back 5 icons
* `AudioButton`s to close to each other in selects
    * Use `@dimen/touch_target_size` for button sizes (`48dp`)

This also addresses the problems found in #3123.

Here's some before screenshots:

Screenshot_1569855511

Screenshot_1569855529

And some after screenshots:

Screenshot_1569855213

Screenshot_1569855236

#### What has been done to verify that this works as intended?

Run in dark and light mode and played with differnet media forms.

#### Why is this the best possible solution? Were any other approaches considered?

For this kind of thing there are of course a lot of different ways we could go. I tried to go with the most minimal changes I could to fix each of the issues we've identified. It'd be good to get others opinions on if there are things they think could be better with these changes.

Another thing: I definitely had to hold off on making changes to the layout of the whole `AudioVideoImageTextLabel` so the spacing feels a little off still as the positioning of the text itself is top left. Changing this will change the size of all the widget labels and selects however so I wanted to avoid it in this PR. After we're happy with this we should create an issue to try and revise that spacing from the perspective of it looking nicer but also potentially increasing the tap target area for selects.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

Hopefully there shouldn't be any regressions as it's just spacing and iconography changes but it'd be good to test with media forms (such as [Birds.zip](https://github.com/opendatakit/collect/files/3672322/Birds.zip)) to see if there is any useability problems on different screen sizes/devices etc.

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/opendatakit/collect/blob/master/CONTRIBUTING.md#ui-components-style-guidelines)

### CODE REVIEW

**Comment by seadowg** [comment]
I'm going to have a play with trying to make the icons more "tappable" after both myself and @lognaturel seemed to feel they weren't.

**Comment by seadowg** [comment]
Here's screenshots of a rework that uses Material's [Outlined Button](https://material.io/components/buttons/#outlined-button). The description for this component ("They contain actions that are important, but aren’t the primary action in an app") felt about right for what we're trying to achieve and I like what it's ended up looking like:

Screenshot_1570551810

Screenshot_1570551839

Again, the margins and positioning of the buttons isn't the best but I think we want to look at that with the whole top label in mind so we don't end up with lopsided margins on right to left layouts etc.

**Review by lognaturel — APPROVED** [review]

**Comment by lognaturel** [comment]
Thanks! The outline buttons work really well, I think.

Here's what I like about this:
- There's greater differentiation between audio and video icons but the two icons are similar to the ones they're replacing so there's little chance that users currently leveraging the features will be confused
- The outlined icons are clearly tappable and don't introduce a new visual concept (e.g. round buttons)
- There's sufficient vertical separation between icons when choices have media

From a process perspective, I think getting outside designer consultation was valuable.

In summary, I'm comfortable that this will be an overall improvement that can be further iterated on and I think it'd be really great to get into v1.24. I hope that you'll agree, @grzesiek2010.

@grzesiek2010 if you're happy with both this and the analytics PR, you may consider merging the analytics PR without QA and then QA both at the same time.

**Comment by grzesiek2010** [comment]
> I hope that you'll agree, @grzesiek2010 

I don't like the fact that we are adding a lot of new prs to a release just before the code freeze it's something we are doing constantly, but ok it's something we should talk about later (after this release).

**Comment by kkrawczyk123** [comment]
@seadowg I am not sure what do you mean by "Prompt is viewed that has a audio media attached to it's label (Prompt/AudioLabel)" do you have any example?

**Comment by seadowg** [comment]
@kkrawczyk123 Yeah sorry that's not the best description. I mean that a question has audio media (using `media::audio`) and that in that situation it will send the event "Prompt":"AudioLabel":"<formID>".

**Comment by kkrawczyk123** [comment]
@seadowg firstly, I am a huge fan of this new design of the buttons!! :)
Verified on Androids 4.2, 4.4, 5.1, 6.0, 7.1, 8.1 and 9.0 dark and light mode both.
 
I have noticed that there is no space between question/choices and audio/video buttons in RTL languages, see attached screenshots:
Screenshot from 2019-10-09 12-38-52
Screenshot from 2019-10-09 12-38-02
As it is not a high-priority issue, especially before release I can open a new issue for improving that, please let me know what do you think about that. 

Tracking events confirmed: AudioLabel, AutoplayAudioLabel, AutoplayAudioChoice, AudioChoice and AudioChoiceGrid on Android 8.1.

@opendatakit-bot unlabel "needs testing"
@opendatakit-bot label "behavior verified"

**Comment by seadowg** [comment]
@kkrawczyk123 hmmm interesting. Is there any problem tapping the buttons because of this? If not, I'd vote for us leaving it as I think we want to revise the spacing/layout for the prompts and select choices after this to make them a bit more cosmetically pleasing.

**Comment by kkrawczyk123** [comment]
@seadowg there is no functioning problem, just a UI this is why I am not complaining that much xd

**Comment by grzesiek2010** [comment]
I agree we can leave it as is.

**Comment by seadowg** [comment]
Posted about follow up work in #3382
